ethereumkit/src/main/java/org/p2p/ethereumkit/external/balance/BalanceRepository.kt

```diff
@@ -1,12 +1,11 @@
 package org.p2p.ethereumkit.external.balance
 
-import com.google.gson.JsonElement
+import java.math.BigInteger
 import org.p2p.ethereumkit.external.api.response.TokenBalancesResponse
 import org.p2p.ethereumkit.external.api.response.TokenMetadataResponse
 import org.p2p.ethereumkit.internal.models.EthAddress
-import java.math.BigInteger
 
-internal interface BalanceRepository {
+interface BalanceRepository {
```

Comment:
i think you are getting balance through EthereumKitRepository, why do you remove internal?
if it's for tests - just write tests in this module, this is not a reason to break incapsulation :(